swagger/swagger-generator/generator-core/src/test/java/org/apache/servicecomb/swagger/generator/core/TestParamUtils.java

```diff
@@ -95,4 +104,34 @@ public void setParameterTypeByTypeOnComplexType() {
           + "param name is [testName]", e.getMessage());
     }
   }
+
+  private static class AllTypeTest {
+    TestType1 t1;
+
+    TestType2 t2;
+
+    List<TestType1> t3;
+
+    Map<String, TestType2> t4;
+
+    Map<TestType1, TestType2> t5;
+  }
+
+  @Test
+  public void testAddDefinitions() {
```

Comment:
expect not throw exception?
just generate swagger, and check the result?

Reply:
considering coexists with previous demo,only print log

Reply:
better to check the result.